.github/workflows/autogpt-builder-docker.yml

```diff
@@ -0,0 +1,41 @@
+name: AutoGPT Builder Docker
+
+on:
+  push:
+    branches: [ master ]
+    paths:
+      - 'rnd/autogpt_builder/**'
+
+defaults:
+  run:
+    shell: bash
+    working-directory: rnd/autogpt_builder
+
+env:
+  PROJECT_ID: agpt-dev
```

Comment:
sticking to dev for now. when we have users, prod and a release agreement we can move to prod vs dev